src/components/Entry.vue

```diff
@@ -158,31 +178,70 @@ export default {
       return (
         !this.isMultipleIDEntryAllowed &&
         !this.isCurrentEntryIncomplete &&
-        this.numOfUserIds < NUMBER_OF_INPUTS_ALLOWED
+        this.numOfUserIds < this.maxNumberOfInput
       );
     },
+
     /** Checks if the current input entry has the required number of characters */
     isCurrentEntryIncomplete() {
-      return this.latestEntry["userID"].length < this.maxLengthOfSRN;
+      return this.latestEntry["userID"].length < this.maxLengthOfId;
     },
+
     /** Returns the most recently entered input */
     latestEntry() {
       return this.userIDList.slice(-1)[0];
     },
+
     /** Whether the current typed ID is valid */
     isInvalidLoginMessageShown() {
       return !this.isCurrentUserValid && this.validateCount == 1;
     },
+
     /** Whether input being typed is in the correct format */
     isInvalidInputMessageShown() {
       return this.invalidInputMessage != null;
     },
+
+    /** Returns the heading text for the input box */
+    inputBoxDisplayText() {
+      return this.programData.text.default.display;
+    },
+
+    /** Returns the button text for adding another input box */
+    inputBoxAddButtonText() {
+      return this.programData.text.default.addButton;
+    },
+
+    /** Returns the maximum length of the ID */
+    maxLengthOfId() {
+      return this.programData.input.maxLengthOfId;
+    },
+
+    /** Returns the basic validation type for the input */
+    basicValidationType() {
+      return this.programData.input.basicValidationType;
+    },
+
+    /** Returns the invalid input message stored against each program */
+    invalidInputText() {
+      return this.programData.text.default.invalid.input;
+    },
+
+    /** Returns the maximum number of ID's a user can enter */
+    maxNumberOfInput() {
```

Comment:
Why not name the method to `maxNumberOfIds` -- both comment and variable name seem to indicate this